rpcs3/Emu/Cell/lv2/sys_rsx.cpp

```diff
@@ -203,9 +203,10 @@ error_code sys_rsx_context_allocate(vm::ptr<u32> context_id, vm::ptr<u64> lpar_d
 
 	for (int i = 0; i < 256; ++i)
 	{
-		reports.semaphore[i].val = 0x1337C0D3;
-		reports.semaphore[i].zero = 0x1337BABE;
-		reports.semaphore[i].zero2 = 0x1337BEEF1337F001;
+		reports.semaphore[i + 0].val.raw() = 0x1337C0D3;
```

Comment:
you can't use i...i+n notation when the loop increment is one!

Reply:
Fixed.